recipes/bsp/grub2.yaml

```
@@ -1,4 +1,7 @@
-inherit: [autotools, autoconf]
+inherit: [autotools, autoconf, "basement::update-config"]
+
+privateEnvironment:
+    APPLY_UPDATE_CONFIG: "$(if-then-else,$(eq,${GRUB2_BOOTSTRAP},1),no,yes)"
```

```suggestion
    APPLY_UPDATE_CONFIG: "$(if-then-else,${GRUB2_BOOTSTRAP:-0},no,yes)"
```

Because `GRUB2_BOOTSTRAP` can only be `0` or `1`, it can be fed directly into `if-then-else`. But it might be unset, therefore the `:-0` to force a default value. BTW, another possibility is to add a default value to the `Config` entry (made possible since [this commit in basement](https://github.com/BobBuildTool/basement/commit/aa16298d1e1106b4d54e47d0b3a980b8c25f770f)).

---

Ah yes thanks @jkloetzke, I already prepared this fix but forgot to push it to the branch :/ Will update soon.

Default values for `Config` are quite useful, thanks.